app/frontend/src/components/forms/FormSubmission.vue

```diff
@@ -175,6 +176,14 @@ export default {
   computed: mapGetters('form', ['form', 'formSubmission', 'permissions']),
   methods: {
     ...mapActions('form', ['fetchSubmission', 'getFormPermissionsForUser']),
+    deleted() {
```

Comment:
Optional: May be better to change this to be something like "onDelete" to better reflect that this is in response to an action.

Reply:
Yeah that's a good idea. Using `onX` in emitted event handlers by the parent would be a good pattern